components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F

```diff
@@ -1160,11 +1170,10 @@ subroutine compute_HJ99_melt_fluxes( &
       Tlatent = latent_heat_fusion_mks/cp_sw
 
       eta = cpRatio * transferVelocityRatio
-      TlatentStar = Tlatent + cpRatio*(T0-iceTemperature(iCell))
       a = -dTf_dS*(1.0_RKIND - eta)
-      b = (transferVelocityRatio*TlatentStar - eta*dTf_dS*oceanSalinity(iCell) &
+      b = (transferVelocityRatio*Tlatent - eta*dTf_dS*oceanSalinity(iCell) &
         + oceanTemperature(iCell) - T0)
-      c = -transferVelocityRatio*TlatentStar
+      c = -transferVelocityRatio*Tlatent + landIceConductiveHeatFlux(iCell) / (rho_sw * oceanHeatTransferVelocity(iCell))
```

Comment:
```suggestion
      c = -transferVelocityRatio*Tlatent - landIceConductiveHeatFlux(iCell) / (rho_sw * oceanHeatTransferVelocity(iCell))
```
I think I actually need to change the sign here since heat fluxes are negative out of the ocean. I'll verify this before committing this change.

Reply:
Like above, I think the units of the 2 terms in `c` are not the same.  We need to work that out.